src/style/app.module.css

```diff
@@ -1931,11 +1987,13 @@ hr {
 }
 
 .toggleBtn:hover {
-  color: var(--light-blue) !important;
+  color: var(--brown-color) !important;
+  border: 1px solid var(--input-shadow) !important;
 }
 
 input[type='radio']:checked + label {
-  background-color: var(--brand-primary-light) !important;
+  color: var(--black-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`